doc/maintaining/data-viewer.rst

```diff
@@ -1,223 +1,332 @@
-===========
-Data Viewer
-===========
+==============================
+Data preview and visualization
+==============================
 
+.. versionchanged:: 2.3
 
-.. todo:: This whole page is outdated and needs to be updated for the new Resource Views
+    The whole way resource previews are handled was changed on CKAN 2.3.
+    Please refer to previous versions of the documentation if you are using
+    an older CKAN version.
 
+.. contents::
 
-The CKAN resource page can contain a preview of the resource's data.
-This works by either:
 
-1. Embedding the data into the page, either directly or by loading the data
-   in an iframe.
-2. Using a custom widget (such as `Recline <http://okfnlabs.org/recline>`_)
-   to view the data.
 
-Generally, the decision as to which action to take is determined by the type of
-resource being viewed.
-In general, images will be directly embedded, unstructured or plain text
-files will be loaded in an iframe, and more complex data types will need to
-use a custom widget.
+Overview
+--------
 
-The data preview functionality that is provided by CKAN is described in
-the following sections:
+The CKAN resource page can contain one or more views of the resource data
```

Comment:
I would say "one or more visualizations" then put examples in brackets (a bar chart, a table, etc). Then go on to say "These are commonly referred to as *resource views*". I think you have to explain it without using the term *resource view* first, then once you've introduced the term you can use it from then on. The reason is that *resource view* is a CKAN jargon term, will not mean anything to people at first.